qcodes/configuration/qcodesrc_schema.json

```diff
@@ -214,7 +214,6 @@
                             "description": "Upper and lower percentage of datapoints that may maximally be discarded by the auto color scale. For example for [1,1] the auto color scale will in a set of 10 0000 points not clip more than the 100 points of lowest and highest value.",
                             "type": "array",
                             "items": {
-                                "type": "number",
```

Comment:
I suggest to revert this and go with the change in https://github.com/QCoDeS/Qcodes/pull/3097

Reply:
Good catch. I was just following pylint gving it as an error. My bad.

Reply:
No worries the original syntax is wrong so this fix is better than the original file but #3097 is even better